lib/src/utils/json_helper.dart

```diff
@@ -191,19 +192,22 @@ class JsonHelper {
         continue;
       }
       final Map<String, dynamic> item = <String, dynamic>{};
-      item['sizes'] = <String, Map<String, int>>{};
+      item['sizes'] = <String, Map<String, dynamic>>{};
```

Comment:
`dynamic` implies the object can be null.
Is-it the case?

Reply:
It cannot be `null` but now it can also be a `String` (for `url`) (and not only `int` for sizes).
Would you prefer `Object` to `dynamic` or is `dynamic` finally good enough?

Reply:
My point here was indeed that if it can't be null, please use an `Object` instead.

Reply:
> My point here was indeed that if it can't be null, please use an `Object` instead.

Good point!